### ISSUE TITLE
Consider dropping md2po, direct use of md files by Weblate

### ISSUE THREAD

**Issue by keunes** [AntennaPod/antennapod.github.io#325]
**Short description**: Weblate now supports md files. It might be worth feeding Weblate with markdown files directly, without having to go through md2po.

**Location**: everywhere

**Why have this**: Dropping one link in the translations chain might make things less complex.

**More info**: https://docs.weblate.org/en/weblate-5.0/formats/markdown.html

**Comment by ByteHamster** [comment]
I think Weblate only supports one file per translation component. We would therefore have to add every single file manually. This clutters the Weblate interface and is a lot more work when creating a new documentation page.

**Comment by keunes** [comment]
One component can cover/include multiple files (see https://docs.weblate.org/en/latest/admin/projects.html#component-filemask)

**Comment by keunes** [comment]
Right, that's true. Although you can set up component discovery, so that you don't have to manually add each file on Weblate: https://docs.weblate.org/en/latest/admin/addons.html#addon-weblate-discovery-discovery

(Nested) categories would allow us to group component files, e.g. all documentation, as we have now: https://docs.weblate.org/en/latest/admin/projects.html#category

**Comment by ByteHamster** [comment]
Even for translators, having multiple components could be a bit hard to get an overview of (translation progress bar is then per file, etc). I already (without dozens of components) find Weblate to be quite a bit more cluttered than Transifex. Also, we would no longer be able to ignore specific sections in the files (using md2po-disable comments).

Unless md2po causes issues, I have a slight preference to keep it as it is for now.

**Comment by keunes** [comment]
> Even for translators, having multiple components could be a bit hard to get an overview of (translation progress bar is then per file, etc).

Weblate introduced categories last year. I've now set them up for website and app store. These categories act like meta-components: you can see their progress and translate them. They can be nested, so we could add 'documentation' under Website.

You can see here the two categories I set up: https://hosted.weblate.org/projects/antennapod/

**Comment by ByteHamster** [comment]
This is nice if translators click on the project page itself. If they use the dashboard (main page, https://hosted.weblate.org/), it still displays every component individually. Also, you can't start translating a category, you have to click each individual component. That could lead to translators missing files.

**Comment by keunes** [comment]
> If they use the dashboard (main page, https://hosted.weblate.org/), it still displays every component individually.

Right. I never use the dashboard (I always go to the project) but that's not convenient indeed. I'll create an issue requesting improvement there.

> Also, you can't start translating a category, you have to click each individual component.

That is actually possible. See for example https://hosted.weblate.org/translate/antennapod/app-store-listing/-/de/

To get there you click category → Languages → pencil. But I agree it's not very clear and a 'translate' button on category level would be welcome. I'll create an issue to request this.
